mPowerSDKTests/MockAPCDataGroupsManager.h

```diff
@@ -0,0 +1,25 @@
+//
+//  MockAPCDataGroupsManager.h
+//  mPowerSDK
+//
+//  Created by Shannon Young on 1/19/16.
+//  Copyright © 2016 Sage Bionetworks. All rights reserved.
```

Comment:
(maybe not strictly necessary in test-related files... can't remember whether I ever added those in BridgeSDK)